spec/rspec/core/option_parser_spec.rb

```diff
@@ -317,6 +340,24 @@ def generate_help_text
       end
     end
 
+    describe "--bisect" do
+      it "sets the `:bisect` option" do
+        options = Parser.parse(%w[ --bisect ])
+
+        expect(options[:bisect]).to be_truthy
+      end
+
+      it "sets the `:runner` option with the `bisect` invocation" do
+        bisect = double(:bisect)
+        allow(RSpec::Core::Invocations).to receive(:bisect).and_return(bisect)
```

Comment:
> The downside there is it doesn't let you capture extra info in the callable (as required by .print_help).

I missed that little detail.  Thanks for pointing it out!

> Maybe we could turn them into callable objects?

:+1:

I like this approach a lot.